native/machine-coding/Quiz-app-with-timer/index.css

```diff
@@ -0,0 +1,109 @@
+@import url(../../helpers/core.css);
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
```

Comment:
no need to add the * ruleset as it is covered in core.css

Reply:
Okay i will change it , should i create a new pull request? I am new to github